### DISCUSSION TITLE
Debian repository

### DISCUSSION THREAD

**Discussion by frague59** [dbeaver/dbeaver#17292] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Provide a debian / ubuntu repository for dbeaver-ce / dbeaver. 

**Describe the solution you'd like**
Provide a debian repository for downloading new releases of dbeaver / dbeaver-ce, so we could user `apt install dbeaver-ce` / `apt update ` / `apt upgrade` to install and upgrade automatically.

**Comment by Matvey16** [comment, marked as answer]
@frague59 We currently have repositories to use on debian / ubuntu for easier updates. Instructions on how to use them can be found on https://dbeaver.io/download/

![image](https://user-images.githubusercontent.com/82543000/183428708-8cebe830-03b5-41d6-a081-407981469196.png)

**Reply by frague59** [reply]
Hi, Thanks for your answer -- I didn't see it - Sorry for the noise...

**Reply by KaKi87** [reply]
Hello,

What about for the enterprise edition ?

Thanks

**Reply by Matvey16** [reply]
Currently any PRO version (Enterprise/Ultimate/Lite) for Linux can only be installed by directly downloading DEB/RPM or .tar.gz files from our website.

**Reply by KaKi87** [reply]
When will those be added to the repository ?

Thanks

**Reply by zeliboba7** [reply]
Is this repo still maintained? 
Accordingly to  https://web.archive.org/web/20260227182641/https://dbeaver.io/download/ the latest snapshot with these instructions was available on 27 Feb 2026 with dbeaver-ce version 25.3.5, but they disappeared on 03 Mar 2026 with version 26.0.0

**Reply by KaKi87** [reply]
The instructions are still there :

<img width="920" height="498" alt="Capture d’écran 2026-06-26 à 14 24 49" src="https://github.com/user-attachments/assets/5a0b2105-4ae9-452e-874f-89ce92b0586b" />
<img width="911" height="465" alt="Capture d’écran 2026-06-26 à 14 24 58" src="https://github.com/user-attachments/assets/3574b246-9eff-486a-937b-6e259c52d2fb" />

As for the Enterprise edition, some time ago, I started developing a tool named [dynapt](https://git.kaki87.net/KaKi87/dynapt) to automatically fetch updates through APT for packages that don't provide an APT repository.

As it's still experimental, you can download the latest pre-release of it and set it up from [this tutorial](https://gist.github.com/KaKi87/f620788e9901abbfef4978eb7ad358b4).

For DBeaver specifically, replace the following configuration line :

```json
    "apps": []
```

With the following :

```json
    "apps": [
        {
            "name": "DBeaver Enterprise",
            "url": "https://dbeaver.com/files/dbeaver-ee-latest-linux-x86_64.deb"
        }
    ]
```

Replace `x86_64` with `aarch64` if applicable.

For questions or issues, please submit a comment on the tutorial page.

Enjoy !
